frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -1,38 +1,52 @@
 import React, { useState } from 'react';
 import map from 'lodash/map';
+import * as moment from 'moment';
 import { createComment } from '../../lib/apiClient';
 import './RecipeComments.css';
 
-const RecipeComments = ({ comments, recipeId, user }) => {
+const RecipeComments = ({
+  comments, recipeId, user, reRenderComments,
+}) => {
   const [comment, setComment] = useState('');
 
   const handleChange = (event) => {
     event.preventDefault();
     setComment(event.target.value);
   };
 
-  const handleSubmit = () => {
-    createComment(comment, recipeId);
+  const submitComment = async () => {
+    setComment('');
+    await createComment(comment, recipeId);
```

Comment:
Didn't we just set `comment` to an empty string?

Reply:
setComment('') resets the comment form/text area. Without that bit, the form does not reset.

Reply:
Right. And I know this works due to the asynchronous nature of `setComment`. It's just somewhat counterintuitive to set the `comment` to an empty string, then rely on its previous value on the next line, if that makes sense.

Reply:
I think it would still work if I move the setComment down to after reloadComments();

Would that address/fix this?

Reply:
I went ahead and moved that setComment down below reloadComments. Does that sufficiently improve the logic/flow of this?